src/gstreamer/KvsSinkStreamCallbackProvider.cpp

```diff
@@ -57,4 +57,18 @@ KvsSinkStreamCallbackProvider::streamClosedHandler(UINT64 custom_data,
     return STATUS_SUCCESS;
 }
 
+STATUS
+KvsSinkStreamCallbackProvider::fragmentAckHandler(UINT64 custom_data,
```

Comment:
Can we change the name to `fragmentAckReceivedHandler` that is what this is called everywhere else.